event_management/src/main/java/com/example/event_management/model/Events.java

```diff
@@ -0,0 +1,46 @@
+package com.example.event_management.model;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotBlank;
+import lombok.*;
+
+@Entity
+@Table(name = "events")
+@Getter
+@Setter
+@ToString
+@NoArgsConstructor
+@AllArgsConstructor
+public class Events {
+
+    @Id
+    @Column(name = "id")
+    private String id;
+
+    @Column(name = "name")
+    @NotBlank(message = "Add Event name!")
+    private String name;
+
+    @Column(name = "date")
+    @NotBlank(message = "Add Date!")
+    private String date;
```

Comment:
This field can store both date and time both.
```suggestion
    @Column(name = "date")
    @NotNull(message = "'dateTime' should not null")
    private LocalDateTime dateTime;
```

Reply:
updated to LocalDate as it is kept separate in Frontend